ivy_tests/test_ivy/test_frontends/test_tensorflow/test_random.py

```diff
@@ -373,3 +373,117 @@ def test_tensorflow_gamma(
         seed=seed,
         test_values=False,
     )
+
+
+@handle_frontend_test(
+    fn_tree="tensorflow.random.stateless_binomial",
+    output_dtype=helpers.array_dtypes(
+        available_dtypes=("float32", "float64"),
+    ),
+    shape=helpers.get_shape(
+        allow_none=False,
+        min_num_dims=1,
+        max_num_dims=3,
+        min_dim_size=1,
+        max_dim_size=5,
+    ),
+    counts=st.one_of(
+        helpers.ints(allow_inf=False, allow_nan=False, min_value=-2, max_value=5),
```

Comment:
these arguments are not supported by `helpers.ints`
```suggestion
        helpers.ints(min_value=-2, max_value=5),
```